modules/tinymce/tools/tasks/emojis.js

```diff
@@ -0,0 +1,48 @@
+const resolve = require('resolve');
+const twemoji = require('twemoji');
+
+const prettyPrint = (obj) => JSON.stringify(obj,null, 2);
+
+const generateContent = (json, attribution) => {
+  const header = 'window.tinymce.Resource.add(\'tinymce.plugins.emoticons\', ';
+  const footer = ');';
+  return '// Source: npm package: emojilib' + attribution + '\n' + header + json + footer;
```

Comment:
1. Do we need to do anything to properly escape `json`
2. Is this file large enough that we should do something like this instead: https://v8.dev/blog/cost-of-javascript-2019#json

Reply:
1. No
2. This is how it was previously done and it's had no performance issues, so I'm going with no (its less than 500kb).